styles/designs/cardboard/parts/_exercises-components.scss

```diff
@@ -115,14 +115,53 @@ $InjectedExercise__Question__Stem--WithFirstElement: (
     )
 );
 
-$InjectedExercise__Question__Answers--WithPre: (
+$InjectedExercise__Question__MultiChoiceAnswers: (
+  _name: "InjectedQuestionAnswers",
+  _subselector: " ol[data-type=question-answers][type=a]",
+  _properties: (
+    counter-reset: multi-choice-index,
+    padding: 0,
+    margin: 0
+  )
+);
+
+$InjectedExercise__Question__Answers--AnswerWithPre: (
   _name: "QuestionAnswersWithPre",
   _subselector: " li[data-type=question-answer] pre",
   _properties: (
     margin: 0,
   )
 );
 
+$InjectedExercise__Question__MultiChoiceAnswers--Answer: (
+  _name: "InjectedQuestionAnswersAnswer",
+  _subselector: " > li[data-type=question-answer]",
+  _properties: (
+    display: table,
+    counter-increment: multi-choice-index,
+  )
+);
+
+$InjectedExercise__Question__MultiChoiceAnswers--AnswerLetter: (
+  _name: "InjectedQuestionAnswersAnswerLetter",
+  _subselector: " > li[data-type=question-answer]::before",
+  _properties: (
+    font-weight: bold,
+    color: enum('ValueSet:::REQUIRED'),
+    content: 'counter(multi-choice-index, lower-latin) "."',
```

Comment:
I'm not very familiar with this repo, so just trying to wrap my head around how stuff works - if we only need to bold the markup, why do we need to recreate the counters?

Reply:
That's a good question. While this could potentially be as easy as setting `ol[type=a] > li::marker` font-weight, this PR is part of a saga of making all other lists look more like multi-part questions (which are depicted in the `Critical Thinking` part of the PDF screenshot). There have been four other issues going in this direction and they always ask for one thing at a time.

Since those are styled in a very specific way that I could not replicated using the `::marker` pseudo element, I recreated the list with `::before` which is more flexible.

I should probably have a screenshot of what these looked like before in the PR (will add).